app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/PinTileTests.java

```diff
@@ -63,8 +66,21 @@ public void tearDown() {
     public void unpinTileFromContextMenu() {
         onView(allOf(withId(R.id.navUrlInput), isDisplayed(), hasFocus()));
 
-        mDevice.pressDPadDown();
-        mDevice.pressDPadDown();
+        Locale currentLocale = LocaleManager.getInstance()
+                .getCurrentLocale(mActivityTestRule.getActivity());
+
+        boolean pocketIsDisplayed = currentLocale.getLanguage().equals("en");
```

Comment:
nit: instead of branching on locale (which duplicates logic), maybe we should branch based on whether or not the view is visible. Probably not worth your time: add a comment for when the test needs to change next, maybe?

Reply:
I'm sure it's possible, but I don't know how to do that in Espresso.  I'll add a comment.